packages/dev/core/src/Shaders/ShadersInclude/pbrBlockAlbedoOpacity.fx

```diff
@@ -81,7 +81,7 @@ albedoOpacityOutParams albedoOpacityBlock(
     #if !defined(SS_LINKREFRACTIONTOTRANSPARENCY) && !defined(ALPHAFRESNEL)
         #ifdef ALPHATEST 
             #if DEBUGMODE != 88
-                if (alpha < ALPHATESTVALUE)
+                if (alpha + 0.0000001 < ALPHATESTVALUE)

```

Comment:
For my understanding, why do we need to add this offset?

Reply:
It's needed to address a quirk in the way shaders work, but we are not planning on fixing it. I've reverted this.